### DISCUSSION TITLE
What is translation key for "Maximum Authentication Age" in Authentication > Password Policy?

### DISCUSSION THREAD

**Discussion by ChetRam** [keycloak/keycloak#29052] [Q&A]
Hi Team,  
We have enabled "Maximum Authentication Age" in Authentication section in Password Policy Tab.
then we have a help icon, which show some help message on click on it.
Currently is show untranslated string (passwordPolicies.maxAuthAge). we would like to translate it using "messages_en.properties" message file.

Please help us to know what is exact key required to translate it.

Please find the screen attached
![maxAuthAgeIssue](https://github.com/keycloak/keycloak/assets/6292813/794086ac-cef2-44a1-b8d7-f85836d9216c)

**Comment by codespearhead** [comment]
I'm not sure I understand the question.

If you follow the [Keycloak 24.0.3's Docker Quickstart Guide](https://www.keycloak.org/getting-started/getting-started-docker), you'll see that the [CSS tooltip](https://www.w3schools.com/css/css_tooltip.asp) is properly displayed.

![Property passwordPoliciesHelp maxAuthAge is displayed correctly](https://github.com/keycloak/keycloak/assets/72931357/da660f73-6df1-4f5d-8184-3d973ee2fec7)

If you're not using the default Keycloak theme, then whatever third-party theme you're using might not support your current Keycloak version yet.

As for you second question, you should do as [this PR](https://github.com/keycloak/keycloak/pull/28962) shows.

https://github.com/keycloak/keycloak/blob/b24d81c846baa53ffb78177cdc3c10f0a20ed84d/js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties#L2117
